satellite/payments/localpayments/local.go

```diff
@@ -107,14 +101,24 @@ func (*service) GetPaymentMethod(ctx context.Context, id []byte) (_ *payments.Pa
 // CreateProjectInvoice creates invoice from provided params
 func (*service) CreateProjectInvoice(ctx context.Context, params payments.CreateProjectInvoiceParams) (_ *payments.Invoice, err error) {
 	defer mon.Task()(&ctx)(&err)
-	// TODO: fill data
-	return &payments.Invoice{}, nil
+	return nil, internalPaymentsErr.New("invoice creation is not allowed with local payments")
 }
 
 // GetInvoice retrieves invoice information from project invoice stamp by invoice id
 // and returns invoice
 func (*service) GetInvoice(ctx context.Context, id []byte) (_ *payments.Invoice, err error) {
 	defer mon.Task()(&ctx)(&err)
-	// TODO: get project invoice stamp by invoice id from the db and fill data
-	return &payments.Invoice{}, nil
+	return nil, internalPaymentsErr.New("invoice creation is not allowed with local payments")
+}
+
+// GetProjectInvoices returns nil invoices slice and nil err
+func (*service) GetProjectInvoices(ctx context.Context, projectID uuid.UUID) (_ []payments.Invoice, err error) {
+	defer mon.Task()(&ctx)(&err)
+	return nil, nil
```

Comment:
maybe add some TODO's not to forget implement it?

Reply:
or panic

Reply:
Nope that’s how it should be